---
permalink: /research/
title: "Research"
excerpt: # ""
author_profile: false
---
### We aim to understand RNA-based gene expression regulation mechanisms and to develop new technologies that can be used in biology and medicine.  
  
## ON-GOING PROJECTS
* Small RNA-based high-throughput screening (coupled with CRISPR systems)  
* Ribonucleases in innate immune responses  
* Transposases and transposable elements for genome engineering  
* (Future) viral noncoding RNAs and their use in RNA therapeutics 
  
    
## PREVIOUS STUDIES  
**A comprehensive model for primary microRNA processing (2019)**  
In the recent work, we tested ~40,000 primary microRNA variants using high-throughput sequencing and elucidated the comprehensive processing rules of DROSHA. Therefore, now we know how DROSHA accurately finds the cleavage site on primary microRNAs, and we can use this information for designing short hairpin RNAs (shRNAs) to regulate genes of interest specifically.  
{% include video id="4ILGq0g3rbw" provider="youtube" %}  
&nbsp;&nbsp;*Reused from Molecular Cell (2019) 73:505, with permission from Elsevier*  
  

**The first crystal structure of DROSHA (2016)**  
MicroRNAs are ~22-nt-long RNAs that regulate the expression of specific mRNAs in diverse biological and pathological contexts. In the canonical pathway, a primary transcript of microRNA is sequentially cleaved by DROSHA and DICER to generate a mature microRNA. Understanding the processing mechanism of DROSHA is crucial because the DROSHA cleavage step determines the target specificity of microRNA. Although DROSHA had been identified as a microRNA biogenesis factor more than a decade ago, the structure of DROSHA was not available because no group had succeeded in the purification of a soluble and homogeneous DROSHA protein. In contrast, by using an optimized human cell expression system and deep knowledge on the biochemical property of DROSHA, we successfully purified the DROSHA protein and solved its crystal structure. The structure enabled us to start the complete understanding of the substrate recognition mechanism of DROSHA. The paper was praised by the Cell referees as “a very welcome addition to the microRNA field’s canon” and “an important milestone.”  
{% include video id="KdyepCiTW54" provider="youtube" %}  
&nbsp;&nbsp;*The Cell Video Abstract for the structure of DROSHA*   
  
**Whole RNA-binding protein repertoire of embryonic stem cells (2013)**  
RNA-binding proteins regulate the gene expression program, and many of them are linked to human diseases. In spite of their biological significance, our understanding of RNA-binding proteins had been limited to proteins with known RNA-binding domains. To reveal the comprehensive repertoire of RNA-binding proteins in embryonic stem cells, We carried out an experiment called “interactome capture” in collaboration with the Matthias Hentze group at EMBL. The interactome capture utilizes UV light to induce covalent crosslinks between RNA and nearby proteins, and the RNA-crosslinked proteins are purified and identified by mass spectrometry. As a result, We reported 555 RNA-binding proteins including a subset of stem cell-specific novel RNA-binding proteins. This paper has been cited >200 times and served as one of the three core datasets for a mammalian RNA-binding protein list (Nature Reviews Genetics 2014, 15:829).  
{% include figure image_path="/assets/images/nsmb.jpg" width="600px" %}  
&nbsp;&nbsp;*From Nature Structural & Molecular Biology (2013) 20:1122*
